tests/std/tests/P0466R5_layout_compatibility_and_pointer_interconvertibility_traits/test.cpp

```diff
@@ -199,7 +197,6 @@ constexpr bool test() {
         ASSERT(!is_corresponding_member(&S5::v1, &S6::v2));
         ASSERT(!is_corresponding_member(&S5::v2, &S6::v1));
         ASSERT(!is_corresponding_member(&S5::v3, &S6::v3));
-        ASSERT(!is_corresponding_member<NS, NS>(&NS::v1, &NS::w1));

```

Comment:
This assertion is directly translated from the example in the standard's [\[meta.member\]](https://eel.is/c++draft/meta.member#example-1). We might need to tell the committee that this example is problematic.